### ISSUE TITLE
NVDA does not read spelling/grammar error in MS Word 2013.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3278]
**Reported by Harich on 2013-06-12 13:08**
When you are spell/grammar checking in MS Word 2013, NVDA does not read the explanation text window. The same applies to other suggestion that spell/grammar checker prompts in MS Word 2013.

**Comment by leonardder** [comment]
Closing in favour of #3410
